experimental/GModule/Cohomology.jl

```diff
@@ -390,6 +403,10 @@ Base.:-(a::Generic.ModuleHomomorphism) = hom(domain(a), codomain(a), -mat(a))
 Base.zero(G::GrpAbFinGen) = G[0]
 Base.:-(M::GrpAbFinGenMap) = hom(domain(M), codomain(M), [-M(g) for g = gens(domain(M))], check = false)
 
+function Oscar.mat(M::FreeModuleHom{FreeMod{QabElem}, FreeMod{QabElem}})
```

Comment:
I think elsewhere this function exists and is called `matrix` (which also fits better in our general naming scheme). So perhaps instead change all uses of `mat` to `matrix`?

Reply:
mat is still used (as of, I think yesterday) in AA? Otherwise, I agree totally.